### ISSUE TITLE
Error w/ inception3 checkpoint command?

### ISSUE THREAD

**Issue by scotshinderman** [tensorflow/models#386]
## Please let us know which model this issue is about (specify the top-level directory)
(models/slim)

While using checkpoint inception_v3 (file and command from the website) and cifar10 data I get a graph error:

> INFO:tensorflow:Fine-tuning from /Users/scot/Desktop/tf/inception_v3.ckpt
> INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors.InvalidArgumentError'>, Assign requires shapes of both tensors to match. lhs shape= [10] rhs shape= [1001]
> 	 [[Node: save/Assign_8 = Assign[T=DT_FLOAT, _class=["loc:@InceptionV3/AuxLogits/Conv2d_2b_1x1/biases"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/cpu:0"](InceptionV3/AuxLogits/Conv2d_2b_1x1/biases, save/restore_slice_8)]]
> Caused by op u'save/Assign_8', defined at:
>   File "train_image_classifier.py", line 585, in <module>
> 

does the exclude/trainable need something else?

--> command for training w/ checkpoint:

> python train_image_classifier.py     --train_dir=/Users/scot/Desktop/tf/training/     --dataset_dir=/Users/scot/Desktop/tf/data/cifar10     --dataset_name=cifar10     --dataset_split_name=train     --model_name=inception_v3     --checkpoint_path=/Users/scot/Desktop/tf/inception_v3.ckpt      --checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits     --trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits 

--> download cifar10 data,  no issues:

> python download_and_convert_data.py \
>     --dataset_name=cifar10 \
>     --dataset_dir=/Users/scot/Desktop/tf/data/cifar10
> 

thanks!

**Comment by sguada** [comment]
Can you try?

python train_image_classifier.py --train_dir=/Users/scot/Desktop/tf/training/ --dataset_dir=/Users/scot/Desktop/tf/data/cifar10 --dataset_name=cifar10 --dataset_split_name=train --model_name=inception_v3 --checkpoint_path=/Users/scot/Desktop/tf/inception_v3.ckpt --checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits --trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits

**Comment by rohitgirdhar** [comment]
https://github.com/tensorflow/models/pull/390

**Comment by scotshinderman** [comment]
that works -- I could see the layout in the graph tab of tensorboard but wasn't able to debug the contents of the checkpoint.   thanks!
